src/main/config/docker-compose.yaml.ctmpl

```diff
@@ -0,0 +1,22 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{$keyname := printf "secret/dsde/%s/firecloud-ui/docker-compose.yaml" $environment}}
+{{with vault $keyname}}
+
+ui:
+  image: {{.Data.ui_image}}
+  ports:
+    - "80:80"
+    - "443:443"
+    {{.Data.ui_additional_ports}}
+  {{.Data.ui_dns}}
+  {{.Data.ui_volumes}}
+  environment:
+    GOOGLE_CLIENT_ID: {{.Data.env_client_id}}
+    ORCH_URL_ROOT: {{.Data.env_orchestration_url}}
```

Comment:
I would say it is simpler now, either stand up the ui alone using this docker file (pointing to the dev instances) or stand up the entire env locally using the master docker compose file in firecloud-environment. It would also be easy to setup a docker compose file for just ui and orchestration.